### PR TITLE
Add bert benchmark

### PR DESCRIPTION
**Pull request by lrq619**
Add bert as a standalone benchmark and also a document on how to write and test standalone benchmark locally.

### CODE REVIEW

File: benchmarks/bert/README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+### Invoke once
+3. Start the function with docker-compose
+   ```bash
+   docker-compose -f ./yamls/docker-compose/dc-bert-python.yaml up
+   ```
+4. In a new terminal, invoke the interface function with grpcurl.
+   ```bash
+   ./tools/bin/grpcurl -plaintext localhost:50000 helloworld.Greeter.SayHello
+   ```
+    This will outputs the min, max and mean inference time of 100 inferences, this may take around 20 seconds.
```
Comment (ustiugov) [inline benchmarks/bert/README.md:22]: I think it would make more sense to invoke once per SayHello

File: benchmarks/bert/docker/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,40 @@
+ARG FROM_IMAGE_NAME=nvcr.io/nvidia/tensorflow:19.10-py3
+FROM ${FROM_IMAGE_NAME}
+
+RUN apt-get update && apt-get install -y pbzip2 pv bzip2 libcurl4 curl
+
+WORKDIR /workspace
+
+# RUN python3 -m pip install torch==1.4.0 onnx==1.6.0 transformers==2.4.0 \
```
Comment (ustiugov) [inline benchmarks/bert/docker/Dockerfile:8]: clean up

File: benchmarks/bert/python/config/user.conf [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# Please set these fields depending on the performance of your system to
```
Comment (ustiugov) [inline benchmarks/bert/python/config/user.conf:1]: please refer to this file in the readme

File: benchmarks/bert/yamls/docker-compose/dc-bert-python.yaml [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+      - --addr=0.0.0.0
+      - --port=50051
+    ports:
+      - target: 50051
+    deploy:
+      resources:
+        reservations:
+          devices:
+            - driver: nvidia
+              count: 2
```
Comment (ustiugov) [inline benchmarks/bert/yamls/docker-compose/dc-bert-python.yaml:21]: if we are to test it, it should work on CPU only.

File: docs/adding_standalone_benchmark.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
+# Adding Standalone Benchmarks in vSwarm
```
Comment (ustiugov) [inline docs/adding_standalone_benchmark.md:1]: please move to a separate PR, ask David for a review. Also, please run by Manyi first.

File: docs/adding_standalone_benchmark.md [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+# commit id is the commit you have made to vSwarm-proto repo
+go get github.com/vhive-serverless/vSwarm-proto@{commit id}
+make all
+```
+
+Now, you have a local relay image that uses your modified version of protoc.
+Remeber **DO NOT** commit changes in `tools/relay` to the remote repo
+
+### 3.Dockerize your benchmark
+This is a case-by-case process, guidelines on how to do this will be given in the furture.
```
Comment (ustiugov) [inline docs/adding_standalone_benchmark.md:58]: please add placeholders for adding CI.

File: docs/adding_standalone_benchmark.md [hunk 7]
```diff
@@ -0,0 +1,58 @@
+# Adding Standalone Benchmarks in vSwarm
+This is a detailed document of how to add a standalone benchmark in vSwarm.
```
Comment (ustiugov) [inline docs/adding_standalone_benchmark.md:2]: not sure why "standalone", is it different for multi-function apps?

**Review by ustiugov — CHANGES_REQUESTED** [review]
Thank you! the code quality is good, well structured. 

- [ ] update the dependabot config
- [ ] add CI for docker-compose and Knative
- [ ] make recommendations on configuration of the benchmark. The default configs should be easy to test in CI (e.g., w/o GPUs, 1 req instead of 100)
- [x] create a PR for **Adding a new benchmark to vSwarm** (rename the file to `adding_new_benchmark.md`)

File: docs/adding_standalone_benchmark.md (hunk 7 above)
Comment (lrq619) [reply docs/adding_standalone_benchmark.md:2]: Yes, this document is just for adding standalone benchmark, because multi-function apps might have different development  procedure, for example, vSwarm-proto only contains protobuf for standalone functions, and multi-function benchmark doesn't rely on the relay docker instance, etc.

**Review by ustiugov — APPROVED** [review]
